src/OVAL/oval_generator.c

```diff
@@ -114,6 +117,16 @@ char *oval_generator_get_timestamp(struct oval_generator *generator)
 	return generator->timestamp;
 }
 
+const char *oval_generator_get_platform_schema_version (struct oval_generator *generator, const char *platform)
+{
+	char *platform_schema_version = oscap_htable_get(generator->platform_schema_versions, platform);
```

Comment:
Yes, exactly, the logic here is that the platform version is implicitly the core version if it was not specified in the OVAL file. The specification says nothing about what to do if some platform extension is used in an OVAL file but its schema version was not specified in that OVAL file.

I decided to do this this way, because then you will not need to use everywhere constructions like this (pseudocode):
```
if (oval_core_schema_version >= 5.11.1)
    get_oval_core_schema_version()
else
    get_oval_platform_schema_version()
```